cryptoki/src/mechanism/mod.rs

```diff
@@ -805,6 +805,9 @@ pub enum Mechanism<'a> {
     AesKeyWrapPad,
     /// AES-GCM mechanism
     AesGcm(aead::GcmParams<'a>),
+    /// AES-GCM mechanism with message based API and parameters
+    /// TODO should we reuse the AesGcm and use Option<> to select parameter?
```

Comment:
I think the TODO should use normal comment (`//`) not to be rendered as docs :thinking: